lib/membrane_opus.ex

```diff
@@ -20,12 +20,9 @@ defmodule Membrane.Opus do
 
   @type t :: %__MODULE__{
           bitrate: bitrate_t,
-          channels: channels_t
+          channels: channels_t,
+          self_delimiting?: boolean
```

Comment:
We have documented what are `bitrate` and `channels` so maybe let's do sth similar with `self_delimiting?`.